tenable/io/v3/was/__init__.py

```diff
@@ -45,3 +49,19 @@ def scans(self):
         :doc:`Scans API <scans>`
         '''
         return ScansAPI(self._api)
+
+    @property
+    def templates(self):
+        '''
+        The interface object for the
+        :doc:`Folders API <folders>`
+        '''
+        return TemplatesAPI(self._api)
+
+    @property
+    def user_templates(self):
+        '''
+        The interface object for the
+        :doc:`Folders API <folders>`
```

Comment:
For consistency, Please update Folders API <folders> to Tenable.io Folders API <folders>